source/connection-monitoring-and-pooling/tests/pool-clear-ready.yml

```diff
@@ -0,0 +1,36 @@
+version: 1
+style: unit
+description: after clear, cannot check out connections until pool ready
+operations:
+  - name: ready
+  - name: checkOut
+  - name: clear
+  - name: start
+    target: thread1
+  - name: checkOut
+    thread: thread1
+  - name: waitForEvent
+    event: ConnectionCheckOutFailed
+    count: 1
+  - name: ready
+  - name: checkOut
+events:
+  - type: ConnectionPoolReady
+    address: 42
+  - type: ConnectionCheckedOut
+    address: 42
+    id: 42
```

Comment:
This should be `connectionId: 42` not `id`.

Reply:
fixed